bin/get-amy.py

```diff
@@ -70,6 +70,9 @@ def main(amy_url, output_file, tags, all_=True):
         del slugs
         del workshops
 
+    # Always sort workshops by the ascending start date.
+    unique_workshops = sorted(unique_workshops, key=lambda w: w['start'])
```

Comment:
Personally I wouldn't worry about it, but if you want I can add sorting by the slug as well.